pkg/ingester/ingester.go

```diff
@@ -302,6 +309,10 @@ samples:
 }
 
 func (i *Ingester) append(ctx context.Context, sample *model.Sample) error {
+	if i.cfg.RejectOldSamples && sample.Timestamp < model.Now().Add(-i.cfg.RejectOldSamplesMaxAge) {
```

Comment:
Could we log something here, so that we can see the OrgID? Or is returning an error sufficient to cause that to be logged?

Reply:
We probably want to record this elsewhere to avoid spamming the ingester logs. We've had problems in the past with the "timestamp out of order" logs. Perhaps we could use something similar to the rate tracking used in `ingester/user_state.go` for tracking rejected sample stats.